src/openrct2/Game.cpp

```diff
@@ -530,11 +528,10 @@ void game_load_init()
     {
         viewport_init_all();
         game_create_windows();
-        mainWindow = window_get_main();
     }
     else
     {
-        mainWindow = window_get_main();
+        auto* mainWindow = window_get_main();
```

Comment:
```suggestion
        const auto* mainWindow = window_get_main();
```
Or just directly feed it into `window_unfollow_sprite()`.